Bento/Adapters/CollectionViewAdapter.swift

```diff
@@ -114,11 +114,15 @@ open class CollectionViewAdapterBase<SectionID: Hashable, ItemID: Hashable>
 
     @objc(collectionView:canPerformAction:forItemAtIndexPath:withSender:)
     open func collectionView(_ collectionView: UICollectionView, canPerformAction action: Selector, forItemAt indexPath: IndexPath?, withSender sender: Any?) -> Bool {
```

Comment:
This should be the one satisfying the conformance since we introduced this to fix a UIKit bug. 😬

Reply:
Does that mean that the introduction of the other one is now satisfying the conformance instead?

Reply:
@andersio out of curiosity, is there a Rdar for that UIKit bug? Only managed to find this [one](https://openradar.appspot.com/31375101)